src/pages/guides/one-key-flow.mdx

```diff
@@ -0,0 +1,548 @@
+  return config;
+};
+```
+
+- Within `package.json` change the scripts field for start, build and test. Instead of `react-scripts` replace it with `react-app-rewired`
+
+<Tabs
+  defaultValue="after"
+  values={[
+    { label: "before", value: "before" },
+    { label: "after", value: "after" },
+]}
+>
+
+<TabItem value="before">
+
+```typescript
+"scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
+},
+```
+
+</TabItem>
+
+<TabItem value="after">
+
+```typescript
+"scripts": {
+    "start": "react-app-rewired start",
+    "build": "react-app-rewired build",
+    "test": "react-app-rewired test",
+    "eject": "react-scripts eject"
+},
+```
+
+</TabItem>
+
+</Tabs>
+
+### Installation
+
+<InstallationWithOneKeyFlowFirebase />
+
+### Initialization
+
+Once installed, your Web3Auth application needs to be initialized. Initialization is a 4 step process where we add all the config details for
+Web3Auth:
+
+1. Instantiation
+2. Configuration of Adapters
+3. Initialization of the Web3Auth
+4. Initialization of the Torus
+
+Please make sure all of this is happening in your application constructor. This makes sure that Web3Auth is initialised when your application starts
+up.
+
+For this guide, we are focussing on the additional Torus libraries which are used alongside Firebase to create the one key flow.
+
+#### Instantiating Web3Auth
+
+<InstantiationWeb3Auth />
+
+#### Initialising the Openlogin Adapter
+
+```js
+const openloginAdapter = new OpenloginAdapter({
+  loginSettings: {
+    mfaLevel: "default",
+  },
+  adapterSettings: {
+    network: "testnet",
+    uxMode: "redirect",
+    loginConfig: {
+      jwt: {
+        name: "Web3Auth One Key Login Flow",
+        verifier,
+        typeOfLogin: "jwt",
+        clientId,
+      },
+    },
+  },
+});
+```
+
+Here, you need to pass over your Web3Auth `clientId` in the adapterSettings object and your Custom Auth `verifierName` in the loginConfig object.
+
+<InitializationWeb3Auth />
+
+##### Initialising Torus
+
+```js
+// Importing the packages
+import Torus from "@toruslabs/torus.js";
+import NodeDetailManager from "@toruslabs/fetch-node-details";
+import { subkey } from "@toruslabs/openlogin-subkey";
+```
+
+The above packages will help us fetching the node details and creating the private key.
+
+```js
+// Configurations for the network
+const network = "https://rpc.ankr.com/eth_ropsten",
+
+// Initializing the Torus instance
+
+const torus = new Torus({
+  enableOneKey: true,
+  network,
+});
+
+// Initializing the NodeDetailManager instance for Ropsten Testnet
+
+const nodeDetailManager = new NodeDetailManager({ network, proxyAddress: NodeDetailManager.PROXY_ADDRESS_ROPSTEN });
+```
+
+##### Initialize JWT Provider and getting the token id
+
+The following code snippet will initialise Firebase SDK and fetch us the token id. This token id is required later to generate the private key.
```

Comment:
"... fetch us the token id. This token id is ... "

It should be ID Token, not the token id.